### PR TITLE
fix: Use exception message if `$redis->getLastError()` returns null

### PR DESCRIPTION
**Pull request by Fahl-Design**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

- Failed in production:
```
TypeError: Exception::__construct(): Argument #1 ($message) must be of type string, null given in /var/www/html/vendor/laminas/laminas-cache-storage-adapter-redis/src/Redis.php:235
```

### CODE REVIEW

**Comment by Ocramius** [comment]
Overall, looks sensible, but needs adjustments, as highlighted by the test suite :D

**Comment by boesing** [comment]
Pretty sure that "too many open files" error is unrelated to these changes. Seems like sth. might've changed in the CI container.

**Comment by Fahl-Design** [comment]
It dosen't look like I can do much about the failing tests
can somebody please re-run the actions first to see if it was "magic fail" ? ^^

**Comment by Ocramius** [comment]
@Fahl-Design I restarted them all

**Comment by boesing** [comment]
FYI: 
I'll work on PHP 8.2 support. I expect the same errors in that PR. If I figure out how to fix these issues, I'll rebase this PR afterwards.

**Comment by tecbird** [comment]
something happening here?

**Comment by Ocramius** [comment]
We'd need to go back to green before considering merges, IMO.

File: psalm-baseline.xml [hunk 1]
```diff
@@ -69,21 +69,7 @@
       <code>exec</code>
       <code>setex</code>
     </PossiblyInvalidMethodCall>
-    <PossiblyNullArgument occurrences="16">
```
Comment (boesing) [inline psalm-baseline.xml:72]: Uh, I like

**Review by boesing — APPROVED** [review]
LGTM! Merging. Will prepare a release now.

**Comment by boesing** [comment]
Thanks, @Fahl-Design

**Pull request merged**